model/atmosphere/dycore/tests/dycore_tests/test_solve_nonhydro.py

```diff
@@ -119,6 +119,8 @@ def test_nonhydro_predictor_step(
     ndyn_substeps,
     caplog,
 ):
+    if (backend is not None) and ("gtfn_cpu" in backend.name):
```

Comment:
It this specifically "gtfn_cpu" or any "gtfn" backend that is an issue with the GridTools compiled backends?  (The datatests are currently not run on GPU as for the CI configuration, we should as a follow up of your work change that and see what goes wrong...).

Reply:
It works for `gtfn_gpu`. I will create the function specifically for `gtfn_cpu`